web/packages/teleport/src/Discover/Database/CreateDatabase/useCreateDatabase.ts

```diff
@@ -59,13 +65,13 @@ export function useCreateDatabase(props: AgentStepProps) {
   //      backend error or network failure)
   const [createdDb, setCreatedDb] = useState<CreateDatabaseRequest>();
 
-  const result = usePoll<DatabaseResource>(
+  const dbSvcPollingResult = usePoll<DatabaseResource>(
```

Comment:
Nit: what is `Svc`?

Reply:
i removed `svc` from name (it meant for `service`, and i would have spelt it out but realized that that is wrong naming, it's just polling for a db, not for a db service)